test/support/factory.ex

```diff
@@ -166,6 +167,72 @@ defmodule Wanda.Factory do
     }
   end
 
+  def completed_with_errors(
```

Comment:
@arbulu89 since you are taking care of the execution factory you might want to look at this

Reply:
@nelsonkopliku I don't see this function as `completed_with_errors` as a transforming funciton.
Basically, it creates a totally new one from scratch, using simply the execution_id and group_id. 
I would simply have normal factory function for that.